kim_cnn/get_data.sh

```diff
@@ -0,0 +1,7 @@
+mkdir data
```

Comment:
Please make sure this file is executable (use chmod to add it if it is not). Also add shebang: `#!/usr/bin/env bash`.

Reply:
added